mobile/src/bundles/employer/components/employer-onboarding-form/employer-onboarding-form.tsx

```diff
@@ -0,0 +1,206 @@
+import { EmployerOnboardingFormValidationSchema } from '~/bundles/employer/validation-schemas/validation-schemas';
+
+import { EMPLOYER_ONBOARDING_DEFAULT_VALUES } from './constants/constants';
+import { styles } from './styles';
+
+const locationOptions = Object.values(CountryList);
+
+type Properties = {
+    employerOnboardingData: EmployerOnboardingFormDto | null;
+    onSubmit: () => void;
+};
+
+const EmployerOnboardingForm: React.FC<Properties> = ({
+    employerOnboardingData,
+    onSubmit,
+}) => {
+    const { control, errors, handleSubmit } = useAppForm({
+        defaultValues:
+            employerOnboardingData ?? EMPLOYER_ONBOARDING_DEFAULT_VALUES,
+        validationSchema: EmployerOnboardingFormValidationSchema,
+    });
+
+    const handleFormSubmit = useCallback(() => {
+        //TODO add some logic
+        void handleSubmit(() => {
+            onSubmit();
+        })();
+    }, [handleSubmit, onSubmit]);
+
+    return (
+        <ScrollView
+            contentContainerStyle={[globalStyles.p25, styles.container]}
+        >
+            <Text
+                category={TextCategory.H4}
+                style={[globalStyles.mb15, styles.title]}
+            >
+                Create a profile to find a perfect match to your company
+            </Text>
+            <Text category={TextCategory.H6} style={globalStyles.mb10}>
+                Please, fill out all the fields below, so we could verify your
+                company
+            </Text>
+            <View
+                style={[
+                    globalStyles.width100,
+                    globalStyles.flexDirectionRow,
+                    globalStyles.justifyContentSpaceAround,
+                    globalStyles.mb25,
+                ]}
+            >
+                <View>
+                    <FormField
+                        errorMessage={
+                            errors.profilePhoto && 'Profile photo is required'
+                        }
+                        name="profilePhoto"
+                        containerStyle={globalStyles.alignItemsCenter}
+                    >
+                        <AvatarPicker
+                            isSingleAvatarView
+                            control={control}
+                            name="profilePhoto"
+                        />
+                    </FormField>
+                    <Text category={TextCategory.H6}>Profile photo</Text>
+                </View>
+                <View>
+                    <FormField
+                        errorMessage={
+                            errors.companyLogo && 'Company logo is required'
+                        }
+                        name="companyLogo"
+                        containerStyle={globalStyles.alignItemsCenter}
+                    >
+                        <AvatarPicker
+                            isSingleAvatarView
+                            control={control}
+                            name="companyLogo"
```

Comment:
Let's add a separate ticket for this, to match layout

Reply:
<img width="296" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/b31a9bb7-0768-4e81-8421-210f5745a239">